src/app/example/form/FormExample/FormExample.vue

```diff
@@ -1,134 +1,135 @@
-          required
-          placeholder="Street"
-          v-model="form.street"
-          validation="required"
-          :disabled="addressDisabled"
-        />
-      </vue-grid-item>
-      <vue-grid-item>
-        <vue-input
-          name="zipCode"
-          id="zipCode"
-          required
-          placeholder="Zip code"
-          v-model="form.zipCode"
-          validation="required|integer"
-          :error-message="$t('components.formExample.zipCode.error' /* Please enter a Number */)"
-          :disabled="addressDisabled"
-        />
-      </vue-grid-item>
-    </vue-grid-row>
+      <vue-grid-row>
+        <vue-grid-item>
+          <vue-input
+            name="street"
+            id="street"
+            required
+            placeholder="Street"
+            v-model="form.street"
+            validation="required"
+            :disabled="addressDisabled"
+          />
+        </vue-grid-item>
+        <vue-grid-item>
+          <vue-input
+            name="zipCode"
+            id="zipCode"
+            required
+            placeholder="Zip code"
+            v-model="form.zipCode"
+            validation="required|integer"
+            :error-message="$t('components.formExample.zipCode.error' /* Please enter a Number */)"
+            :disabled="addressDisabled"
+          />
+        </vue-grid-item>
+      </vue-grid-row>
 
-    <vue-grid-row>
-      <vue-grid-item>
-        <vue-input
-          name="city"
-          id="city"
-          required
-          placeholder="City"
-          v-model="form.city"
-          validation="required"
-          :disabled="addressDisabled"
-        />
-      </vue-grid-item>
-      <vue-grid-item>
-        <vue-select
-          placeholder="Choose Country"
-          name="country"
-          id="country"
-          v-model="form.country"
-          :options="countryOptions"
-          validation="required"
-          required
-          :disabled="addressDisabled"
-        />
-      </vue-grid-item>
-    </vue-grid-row>
+      <vue-grid-row>
+        <vue-grid-item>
+          <vue-input
+            name="city"
+            id="city"
+            required
+            placeholder="City"
+            v-model="form.city"
+            validation="required"
+            :disabled="addressDisabled"
+          />
+        </vue-grid-item>
+        <vue-grid-item>
+          <vue-select
+            placeholder="Choose Country"
+            name="country"
+            id="country"
+            v-model="form.country"
+            :options="countryOptions"
+            validation="required"
+            required
+            :disabled="addressDisabled"
+          />
+        </vue-grid-item>
+      </vue-grid-row>
 
-    <vue-grid-row>
-      <vue-grid-item>
-        <vue-checkbox
-          name="acceptTerms"
-          id="acceptTerms"
-          v-model="form.acceptTerms"
-          label="I accept the terms"
-          validation="required"
-          required
```

Comment:
Why don't we validate this checkbox? Seems to be very unintuitive having to check this one separately in the button.

Reply:
I didn't find a way that the validation works correctly, I need to investigate more. The documentation says that "false" should work as "not required" but somehow it doesn't pick it up